dev-libs/efl/efl-1.25.1-r10.ebuild

```diff
@@ -5,7 +5,10 @@ EAPI=7
 
 LUA_COMPAT=( lua5-{1..3} luajit )
 
-inherit lua-single meson xdg-utils
+DOCS_BUILDER="doxygen"
+DOCS_DIR="${S}/doc"
```

Comment:
Does the `${S}` need to get specified now with doxygen? Or is this specific to efl?

Reply:
For sphinx you can get away with using a relative path. For doxygen and mkdocs DOCS_OUTDIR has to be a absolute path because those docbuilders have to be run in the directory where the config file is, which means that if DOCS_DIR is not equal to S (the default) the docbuilder and `einstalldocs` are not run from the same directory which will cause `einstalldocs` to fail because it can't find the directory specified in `HTML_DOCS`. Because DOCS_OUTDIR is set relative to DOCS_DIR this means that DOCS_DIR has to be an absolute path.

This morning I thought of a solution that will allow the eclass to be used with relative paths in DOCS_DIR (changing the default DOCS_OUTDIR from DOCS_DIR/_build/html to S/_build/html). That way DOCS_OUTDIR is an absolute path by default, and it is not longer strictly necessary for DOCS_DIR to be an absolute path as well.